ui/app/AppLayouts/Wallet/controls/FilterComboBox.qml

```diff
@@ -0,0 +1,319 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtGraphicalEffects 1.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls 1.0
+
+import SortFilterProxyModel 0.2
+
+ComboBox {
+    id: root
+
+    required property var regularTokensModel
```

Comment:
Will these be the ones that are NOT part of any collection? Maybe a comment could help to clarify!

Reply:
Yup, will add comments to what is what

Reply:
Fixed